xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.cfg.vm

```diff
@@ -370,6 +370,10 @@ xwiki.authentication.unauthorized_code=200
 #-# Must be set to "empty".
 # xwiki.authentication.createuser=empty
 
+#-# If set to true, you can block the cookies of users from being used except by the same IP address which got them. 
+#-# except by the same IP address which got them.
+# xwiki.authentication.useip=true
```

Comment:
I don't think that the main reason for useip. It seems to me more a side-effect/consequence. Also you have duplicated content.

Reply:
Actually Akshit is right, that's exactly what this property is for: for security reason we restrict the cookie to the same IP by default and it can be disabled with this property.

You should more explicitly indicate what is the default behavior since it feel like it's disabled by default when reading what you wrote. Also you repeat twice "except by the same IP address which got the".

Reply:
I think I'd prefer explaining with more details. I made a proposal in comment earlier, could you check it out?

I'm also not sure that the way it's formulated is clear enough. I don't really understand it personally (could be an issue with English).

Thanks, we're getting closer :)